docs/reference/query-languages/esql/kibana/definition/operators/is_not_null.json

```diff
@@ -0,0 +1,194 @@
+{
+  "comment" : "This is generated by ESQL’s AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.",
+  "type" : "operator",
+  "operator" : "predicates",
+  "name" : "is_not_null",
+  "description" : "Use `IS NOT NULL` to filter data based on whether the field exists or not:",
```

Comment:
```suggestion
  "description" : "Use `IS NOT NULL` to filter data based on whether the field exists or not.",
```

Reply:
Thanks, I've made this change in the source document, so it will be generated here.